apps/src/storage/dataBrowser/data-browser.module.scss

```diff
@@ -1,19 +1,17 @@
 @import 'color.scss';
 
-$tableWidth: 400px;
```

Comment:
I didn't see where some of these variables were being used, so I deleted them all together